roles/cobbler/tasks/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
+- include: ipmi_secrets.yml
```

Comment:
I might move the ``always`` tag here instead of having two of them in ipmi_secrets.yml